.gitignore

```diff
@@ -408,3 +414,6 @@
 
 # /utils/wxrc/
 /utils/wxrc/*.sln
+
+# /include
+/include/**/*.gch
```

Comment:
Do we really need this one? AFAIK only configure builds use PCH and configure builds should really, really be done out of the source tree.